pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -295,19 +291,14 @@ func (exp *flowExporter) sendFlowRecords() error {
 				}
 				exp.numDataSetsSent = exp.numDataSetsSent + 1
 			}
-			// If the connection is not present in the conntrack table anymore,
-			// then we delete the connection in connection store and record store.
-			// Please note that our record cache policy is that we keep the record in
-			// the record map as long as the corresponding connection is present in
-			// the conntrack table.
-			if !record.Conn.IsPresent {
-				klog.V(2).Infof("Deleting the inactive connection with key: %v from both connection map and record map", key)
-				if err := exp.connStore.DeleteConnectionByKey(key); err != nil {
```

Comment:
looks like the following comment in `pkg/agent/flowexporter/connections/connections.go` needs to be updated

```
        // DeleteConnectionByKey deletes the connection in connection map given the
        // connection key. This function is called from Flow Exporter once the connection
        // is deleted from conntrack module.
```